### PR TITLE
Updated index to show Develop Checks topic in the Help ToC

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

**Comment by rasika-chivate** [comment]
Please update the mkdocs-base.yml file as well.

File: mkdocs-base.yml [hunk 1]
```diff
@@ -141,6 +141,8 @@ nav:
       - details/index.md
       - details/architecture.md
       - details/interface.md
+      - Advisors:
+        - details/develop-checks.md
```
Comment (atymchuk) [inline mkdocs-base.yml:145]: I think we need to maintain consistency in indentation

Comment (atymchuk) [inline mkdocs-base.yml:145]:
```suggestion
        - advisors/index.md 
```

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Review by rasika-chivate — APPROVED** [review]

File: docs/details/advisors/index.md [hunk 2]
```diff
@@ -1 +1,15 @@
-# Advisor Documentation
\ No newline at end of file
+# Advisor Documentation
+
+## [How to develop security checks](./checks/develop-checks.md)
```
Comment (atymchuk) [inline docs/details/advisors/index.md:3]: @catalinaadam I'm not really happy with how this one looks like, but couldn't come up with a good alternative. Maybe you can? :)

**Review by atymchuk — APPROVED** [review]

**Pull request merged**